code/src/main/kotlin/com/expediagroup/sdk/core/request/interceptor/HttpRequestLoggingInterceptor.kt

```diff
@@ -19,7 +19,7 @@ import okio.IOException
  */
 class HttpRequestLoggingInterceptor : HttpExecuteInterceptor {
     private val logger: ExpediaGroupLogger =
-        ExpediaGroupLoggerFactory.getLogger(HttpRequestLoggingInterceptor::class.java)
+        ExpediaGroupLoggerFactory.getLogger(com.expediagroup.sdk.core.request.interceptor.HttpRequestLoggingInterceptor::class.java)
```

Comment:
please let's cleanup all fully qualified names usages, we have many in this file